requirements/base.txt

```diff
@@ -7,6 +7,6 @@ future>=0.16.0
 six>=1.11.0
 uritemplate>=3.0.0
 
-djangorestframework>=3.7.0
+djangorestframework>=3.7.4
```

Comment:
I think it should be 3.7.7, since 3.7.4-3.7.6 has problems with missing files from packaging.